src/main/java/yapbot/tasks/Task.java

```diff
@@ -0,0 +1,75 @@
+package yapbot.tasks;
+
+import yapbot.exceptions.YapBotException;
+
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+/**
+ * Parent class of all tasks that can be created for YapBot.
+ */
+public class Task {
+    private String description;
+    private boolean isDone;
+
+    // Formats used for parsing of date and time from user input, locale is explicity set to en_sg for compatibility
+    private static final Locale LOCALE = new Locale("en_sg");
+    protected static final DateTimeFormatter DATETIME_FORMATTER = DateTimeFormatter.ofPattern("ha yyyy/MM/dd",
```

Comment:
I like how you structure the accessibility of local variables (protected, private)